solr/core/src/test/org/apache/solr/search/TestExtendedDismaxParser.java

```diff
@@ -45,6 +45,14 @@
 import org.junit.BeforeClass;
 import org.junit.Test;
 
+import static org.apache.solr.hamcrest.QueryMatchers.booleanQuery;
+import static org.apache.solr.hamcrest.QueryMatchers.boostQuery;
+import static org.apache.solr.hamcrest.QueryMatchers.disjunctionQuery;
+import static org.apache.solr.hamcrest.QueryMatchers.phraseQuery;
+import static org.apache.solr.hamcrest.QueryMatchers.termQuery;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.containsString;
+
 @LuceneTestCase.AwaitsFix(bugUrl = "https://issues.apache.org/jira/browse/SOLR-15389")
```

Comment:
Can we remove the awaits fix now?

Reply:
It should be removed but I'm not done with all the fixes - there's still three more methods in this class that I haven't gotten to yet.